generators/entity-client/templates/angular/src/main/webapp/app/entities/entity-management-update.component.ts.ejs

```diff
@@ -286,6 +293,20 @@ _%>
 <%_ } _%>
         };
     }
+    <%_ for ( idx in fields ) {
```

Comment:
For consistency, please remove the extra spaces around the iteration instruction inside the for

Reply:
ok